packages/typespec-client-generator-core/src/package.ts

```diff
@@ -590,34 +607,38 @@ function getSdkMethods<TServiceOperation extends SdkServiceOperation>(
   }
   for (const operationGroup of listOperationGroups(context, client)) {
     // We create a client accessor for each operation group
-    const operationGroupClient = diagnostics.pipe(
-      createSdkClientType<TServiceOperation>(context, operationGroup, sdkClientType),
+    retval.push(
+      diagnostics.pipe(createClientAccessorMethod(context, operationGroup, sdkClientType)),
     );
-    const clientInitialization = getClientInitialization(context, operationGroup.type);
-    const parameters: SdkParameter[] = [];
-    if (clientInitialization) {
-      for (const property of clientInitialization.properties) {
-        parameters.push(property);
-      }
-    } else {
-    }
-    const name = `get${operationGroup.type.name}`;
-    retval.push({
-      kind: "clientaccessor",
-      parameters,
-      name,
-      doc: getDoc(context.program, operationGroup.type),
-      summary: getSummary(context.program, operationGroup.type),
-      access: "internal",
-      response: operationGroupClient,
-      apiVersions: getAvailableApiVersions(context, operationGroup.type, client.type),
-      crossLanguageDefinitionId: `${getCrossLanguageDefinitionId(context, operationGroup.type)}.${name}`,
-      decorators: [],
-    });
   }
   return diagnostics.wrap(retval);
 }
 
+function createClientAccessorMethod<TServiceOperation extends SdkServiceOperation>(
```

Comment:
just to clarify, we still have client accessor methods, denoting the relationship between a parent and child client. At the same time, instead of treating the initialization information in tcgc as a pure model, we are looking at it more like another operation. I like this way of thinking, I'm just trying to make sure I'm reading this correctly

Reply:
after design review, we decided to keep the original model impl. also, client accessor will be deprecated and switch to `subClients` property.